crates/ruff_linter/src/rules/pydoclint/snapshots/ruff_linter__rules__pydoclint__tests__docstring-extraneous-parameter_DOC102_google.py.snap

```diff
@@ -0,0 +1,182 @@
+---
+source: crates/ruff_linter/src/rules/pydoclint/mod.rs
+---
+DOC102_google.py:6:1: DOC102 `a` is not in the function's signature
+   |
+ 4 |       Adds two numbers and returns the result.
+ 5 |   
+ 6 | /     Args:
+ 7 | |         a (int): The first number to add.
+ 8 | |         b (int): The second number to add.
+ 9 | | 
+10 | |     Returns:
+   | |_^ DOC102
+11 |           int: The sum of the two numbers.
+12 |       """
+   |
+   = help: Remove the extraneous parameter from the docstring
+
+DOC102_google.py:21:1: DOC102 `multiplier` is not in the function's signature
+   |
+19 |       Multiplies each element in a list by a given multiplier.
+20 |   
+21 | /     Args:
+22 | |         lst (list of int): A list of integers.
+23 | |         multiplier (int): The multiplier for each element in the list.
+24 | | 
+25 | |     Returns:
+   | |_^ DOC102
+26 |           list of int: A new list with each element multiplied.
+27 |       """
+   |
+   = help: Remove the extraneous parameter from the docstring
+
+DOC102_google.py:36:1: DOC102 `numbers` is not in the function's signature
+   |
+34 |       Finds the maximum value in a list of numbers.
+35 |   
+36 | /     Args:
+37 | |         numbers (list of int): A list of integers to search through.
```

Comment:
Would you mind adding an example for a multiline argument description that also contains a `:`?

Reply:
I have done it (line 198)